src/ClientData/DataManagerTest.cpp

```diff
@@ -115,6 +116,25 @@ TEST(DataManager, CanOnlyBeUsedFromTheMainThread) {
                                             WineSyscallHandlingMethod::kNoSpecialHandling);
   CallMethodOnDifferentThreadAndExpectDeath(data_manager,
                                             &DataManager::wine_syscall_handling_method);
+  CallMethodOnDifferentThreadAndExpectDeath(data_manager, &DataManager::GetTimeRangeSelectionStart);
+  CallMethodOnDifferentThreadAndExpectDeath(data_manager, &DataManager::GetTimeRangeSelectionEnd);
+  CallMethodOnDifferentThreadAndExpectDeath(data_manager, &DataManager::ClearTimeRangeSelection);
+  CallMethodOnDifferentThreadAndExpectDeath(data_manager, &DataManager::SetTimeRangeSelection, 0,
+                                            5);
+}
+
+TEST(DataManager, TimeRangeSelection) {
+  DataManager data_manager;
+  EXPECT_EQ(data_manager.GetTimeRangeSelectionStart(), std::numeric_limits<uint64_t>::min());
+  EXPECT_EQ(data_manager.GetTimeRangeSelectionEnd(), std::numeric_limits<uint64_t>::max());
+
+  data_manager.SetTimeRangeSelection(5, 10);
+  EXPECT_EQ(data_manager.GetTimeRangeSelectionStart(), 5);
+  EXPECT_EQ(data_manager.GetTimeRangeSelectionEnd(), 10);
+
+  data_manager.ClearTimeRangeSelection();
+  EXPECT_EQ(data_manager.GetTimeRangeSelectionStart(), std::numeric_limits<uint64_t>::min());
+  EXPECT_EQ(data_manager.GetTimeRangeSelectionEnd(), std::numeric_limits<uint64_t>::max());
```

Comment:
Here we duplicate the code testing verifying the interval. 
If we had a `struct TimeRange`, we would write a matcher only once.

Reply:
done.